Packs/ContentManagement/Scripts/CommitFiles/CommitFiles_test.py

```diff
@@ -23,6 +27,79 @@ def test_does_file_exist(mocker):
     assert not flag
 
 
+def test_does_file_exist_azure_devops(mocker):
+    """
+    Given:
+        - A branch name and a content file.
+    When:
+        - In order to know if this file already exists in the repository or not
+    Then:
+        - Returns True if the file exists and false if it doesn't
```

Comment:
i see we only check the case where `flag is False`, what about the other?

Reply:
added another case